dotcom-rendering/src/web/layouts/StandAlonePage.tsx

```diff
@@ -0,0 +1,203 @@
+import { css, Global } from '@emotion/react';
+import {
+	brandAlt,
+	brandBackground,
+	brandBorder,
+	brandLine,
+	focusHalo,
+	neutral,
+	palette,
+} from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import type { ReactNode } from 'react';
+import { StrictMode } from 'react';
+import type { NavType } from '../../model/extract-nav';
+import type { DCRNewslettersPageType } from '../../types/newslettersPage';
+import { AlreadyVisited } from '../components//AlreadyVisited.importable';
+import { FocusStyles } from '../components//FocusStyles.importable';
+import { Footer } from '../components/Footer';
+import { Header } from '../components/Header';
+import { HeaderAdSlot } from '../components/HeaderAdSlot';
+import { Island } from '../components/Island';
+import { Nav } from '../components/Nav/Nav';
+import { ReaderRevenueDev } from '../components/ReaderRevenueDev.importable';
+import { Section } from '../components/Section';
+import { SkipTo } from '../components/SkipTo';
+import { SubNav } from '../components/SubNav.importable';
+import { Stuck } from '../layouts/lib/stickiness';
+import { decideFormat } from '../lib/decideFormat';
+
+type Props = {
+	model: DCRNewslettersPageType;
+	nav: NavType;
+	children: ReactNode;
+};
+
+/**
+ * @description
+ * Article is a high level wrapper for stand alone pages on Dotcom. Sets strict mode and some globals
+ *
+ * */
+export const StandAlonePage = ({ children, model, nav }: Props) => {
+	const {
+		subscribeUrl,
+		editionId,
+		pageFooter,
+		format,
+		contributionsServiceUrl: pageContributionsServiceUrl,
+		config,
+		isAdFreeUser,
+	} = model;
+	const articleFormat: ArticleFormat = decideFormat(format ?? {});
+
+	const renderAds = !isAdFreeUser;
+
+	const isInEuropeTest =
+		config.abTests.europeNetworkFrontVariant === 'variant';
+
+	const contributionsServiceUrl =
+		process.env.SDC_URL ?? pageContributionsServiceUrl;
+
+	return (
+		<StrictMode>
+			<Global
+				styles={css`
+					/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
+					/* The not(.src...) selector is to work with Source's FocusStyleManager. */
+					*:focus {
+						${focusHalo}
+					}
+					::selection {
+						background: ${brandAlt[400]};
+						color: ${neutral[7]};
+					}
+				`}
+			/>
+			<SkipTo id="maincontent" label="Skip to main content" />
+			<SkipTo id="navigation" label="Skip to navigation" />
+
+			<Island clientOnly={true} deferUntil="idle">
+				<AlreadyVisited />
+			</Island>
+			<Island clientOnly={true} deferUntil="idle">
+				<FocusStyles />
+			</Island>
+
+			<Island clientOnly={true} deferUntil="idle">
+				<ReaderRevenueDev shouldHideReaderRevenue={true} />
+			</Island>
+
+			<div data-print-layout="hide" id="bannerandheader">
+				<>
+					{renderAds && (
+						<Stuck>
```

Comment:
This file seems to blend together the `ArticlePage` and `FrontPage` system we have in DCR (which live just in the components directory) with the layouts we have in the layouts directory. It'd definitely be worth looking at this relationship in DCR, if it still makes sense & what is best for these newsletter / standalone pages!